website/docs/language/files/override.mdx

```diff
@@ -150,3 +150,8 @@ In both the `required_version` and `required_providers` settings, each override
 constraint entirely replaces the constraints for the same component in the
 original block. If both the base block and the override block both set
 `required_version` then the constraints in the base block are entirely ignored.
+
+If a `cloud` block is set within the original configuration's `terraform` block and a `backend` block
+is set in the override file, then the backend specified by the `backend` block is used by Terraform upon merging.
+Similarly, if a `backend` block is set within the original configuration's `terraform` block
+and a `cloud` block is set in the override file, then the backend specified by the `backend` block is used by Terraform upon merging.
```

Comment:
```suggestion
The presence of a `backend` block in either the original configuration or an override file always takes precedence over a [Terraform CLI integration `cloud` block](/docs/language/settings/terraform-cloud). If a `cloud` block is set within the original configuration and a `backend` block is set in the override file, Terraform will use the backend specified in the override file upon merging. Similarly, if a `backend` block is set within the original configuration and a `cloud` block is set in the override file, Terraform will use the backend specified in the original configuration upon merging. 
```

Reply:
I like the additional sentence at the beginning, but it raised to me an error in the original documentation suggestion that I wrote. I've fixed that error, while incorporating your suggestion into the beginning.

Reply:
Also kept the write-up changes to occur over multiple lines, in order to match toughly the line lengths in the rest of the documentation.